README.md

```diff
@@ -1,2 +1,66 @@
-# licensespring-obfuscation
-Showcasing how this works
+# licensespring-obfuscation-sample
+
+This is a sample project demonstrating the use of LicenseSpring in a Java application. The project utilizes the LicenseSpring License Client library to manage license activation, checking, and deactivation.
+
+## Prerequisites
+
+- Java Development Kit (JDK) 8 or higher
+- Apache Maven
+
+## pom.xml
+
+The project is built using Maven, and the dependencies are specified in the `pom.xml` file. The main dependencies include:
+
+- `licensespring-license-client`: LicenseSpring License Client library
+- `junit`: A testing framework for Java applications
+
+The `pom.xml` also includes the `maven-assembly-plugin` to generate a JAR file with all dependencies and the `proguard-maven-plugin` to obfuscate the final JAR file.
+
+
+## MyLicenseManager
+
+The `MyLicenseManager` class is a simple wrapper around the LicenseSpring License Client library. It demonstrates license activation, checking, and deactivation.
+
+
+## App (Main Class)
+
+The `App` class is the main entry point of the application. It expects five command-line arguments: `apiKey`, `sharedKey`, `licenceKey`, `productName`, and `isStaging` (yes/no).
+
+```java
+public class App {
+    public static void main(String[] args) {
+        System.out.println("Starting...");
+        if (args.length != 5) {
+            System.out.println("Five arguments required: apiKey, sharedKey, licenceKey, productName and isStaging(yes/no) variable");
+        } else {
+            MyLicenseManager liscenceManagerClass = new MyLicenseManager();
+            liscenceManagerClass.activateAndCheckLicense(args[0], args[1], args[2], args[3], args[4]);
+        }
+    }
+}
+```
+
+## Build
+
+To build the project, run the following command in the project's root directory:
+
+```sh
+mvn clean package
+```
+
+This will generate a JAR file with dependencies and obfuscate the code using ProGuard. The resulting JAR file will be located in the `target` directory.
```

Comment:
put an empty new line after this so the images is in a new line (currently it's wapped around the text)
<img width="229" alt="image" src="https://user-images.githubusercontent.com/33116911/230359636-b0111af3-1898-418a-977c-ed7dcc88cfc3.png">